src/main/java/barry/Storage.java

```diff
@@ -0,0 +1,92 @@
+package barry;
+
+import java.io.*;
```

Comment:
Small issue here, we have to explictly say what we have imported for this project. This might be an issue from the IDE settings. Could go settings > Editor > Code Style > Java > Imports 🤔